staging/src/k8s.io/cli-runtime/pkg/resource/helper.go

```diff
@@ -96,10 +98,18 @@ func (m *Helper) WithSubresource(subresource string) *Helper {
 	return m
 }
 
+// WithResourceVersion sets the helper to request a resource with a specific version
+// (<resource>/[ns/<namespace>/]<name>?ResourceVersion=<resourceVersion>)
+func (m *Helper) WithResourceVersion(resourceVersion string) *Helper {
+	m.ResourceVersion = resourceVersion
+	return m
+}
+
 func (m *Helper) Get(namespace, name string) (runtime.Object, error) {
 	req := m.RESTClient.Get().
 		NamespaceIfScoped(namespace, m.NamespaceScoped).
 		Resource(m.Resource).
+		ResourceVersion(m.ResourceVersion).
```

Comment:
List has different semantics. One very confusing thing about doing this is that the parameter "resource_version" is going to have different meanings when applied to GET as opposed to LIST, where if you specify resource_version, it means "no older than" whereas  in this implementation, it will mean "exactly this RV".

Reply:
/cc @jpbetz @deads2k on the inconsistent semantics between "resource_version" for GET and LIST requests.

I'd prefer using "exact_resource_version" to avoid the inconsistent semantics.